common/app/model/content.scala

```diff
@@ -567,6 +567,7 @@ object Audio {
   }
 
   def acastUrl(player: AudioPlayer, url: String, isAdFree: Boolean): String = if (player.audio.tags.isPodcast && !isAdFree) "https://flex.acast.com/" + url.replace("https://", "") else url
```

Comment:
This method can now make use of the one below it when the condition is true.

Reply:
It's not a problem; just a chance to reduce duplication.

Reply:
I see what you mean, will amend